cql3/Cql.g

```diff
@@ -2214,6 +2216,7 @@ K_ROLES:       R O L E S;
 K_SUPERUSER:   S U P E R U S E R;
 K_NOSUPERUSER: N O S U P E R U S E R;
 K_PASSWORD:    P A S S W O R D;
+K_SALTED_HASH: S A L T E D '_' H A S H;
```

Comment:
It's customary to use separate words in SQL, without underscores. We violated this with SERVICE_LEVEL, but let's not repeat.